modules/migrations/update.go

```diff
@@ -0,0 +1,57 @@
+// Copyright 2019 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package migrations
+
+import (
+	"strconv"
+
+	"code.gitea.io/gitea/models"
+	"code.gitea.io/gitea/modules/log"
+	"code.gitea.io/gitea/modules/structs"
+)
+
+// UpdateMigrationPosterID updates all migrated repositories' issues and comments posterID
+func UpdateMigrationPosterID() {
+	if err := updateMigrationPosterIDByGitService(structs.GithubService); err != nil {
```

Comment:
Is Githubservice OK here? I was under the impression that this function should be service agnostic.

Reply:
Good catch, done.